test_person_rental_book.rb

```diff
@@ -0,0 +1,44 @@
+# Test associations between person, rental and book. Sample code and output below:
+# plese note: This will be removed in the final version of the project as it defined to check ...
+# correct implementation of the associations bewteen person, rental and book.
+require './person'
```

Comment:
- [ ] Just as I mentioned in my previous comment, a similar implementation should be present in this `Person class` as well :+1: 
- [ ]  Per the project requirements, you needed to:

> Create the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental. The following should be implemented:
> Create the has-many side of Person and Rental (a person has many rentals).

- Currently, this isn't being implemented in the `Person class`. I will suggest you include `a method for adding rentals `in the class. The person itself should be sent as a parameter to create the new rental thereby achieving the `has-many` association. :wink: 
- Here is a [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship%20def%20add_rental(person,%20date)) to an example of this implementation in a previous lesson :+1: 
```ruby
  def add_rental(book, date)
     Rental.new(date, book, self)
  end
```